bin/summariser.py

```diff
@@ -171,6 +171,8 @@ def runprocess(db_config_file, config_file):
             log.warning("The summariser may not start again until it is removed.")
 
         log.info(LOG_BREAK)
+    
+    db.db.close()
```

Comment:
Sorry, I can see how you understood me differently. I meant right after the `finally` statement on line 159, rather than after the `finally` block.